### ISSUE TITLE
[Bug]:  Unable to Set Permissions

### ISSUE THREAD

**Issue by toadel** [FTBTeam/FTB-Mods-Issues#670]
### Mod

FTB Ranks

### Mod version

1.16.5 latest

### Forge / Fabric version

1.16.5 latest

### What issue are you having?

I'm using this mod on my server for pixelmon

Unable to set permissions in the file member: {
        name: "Member"
        power: 1
        condition: "always_active"
        ftbranks.name_format: "<{name}>"
        ftbchunks.max_claimed: 50
        "pixelmon.command.checkspawns":true
        "pixelmon.checkspawns.legendary":true
        "pixelmon.command.endbattle":true
        "pixelmon.command.pc":true
        "command.pokepc":true
        command.pokepc:true
    }
this is the config but neither are working

### Steps to reproduce

member: {
        name: "Member"
        power: 1
        condition: "always_active"
        ftbranks.name_format: "<{name}>"
        ftbchunks.max_claimed: 50
        "pixelmon.command.checkspawns":true
        "pixelmon.checkspawns.legendary":true
        "pixelmon.command.endbattle":true
        "pixelmon.command.pc":true
        "command.pokepc":true
        command.pokepc:true
    }
this is the config but neither are working

### Anything else to note?

Unable to set in the config

**Comment by toadel** [comment]
i've also tried a space between : and true

**Comment by desht** [comment]
Apologies for the delay. Can you please paste your entire `ranks.snbt` to a paste site (e.g. pastebin, hastebin, etc.) ?  Lack of formatting in the above makes it hard to know if it's valid.

Additionally, please check your server startup log for errors (or monitor the server log while running `/ftbranks reload`).  Most issues like this are caused by a syntax error in the config file.

**Comment by TRok92** [comment]
Conveniently I'm having the exact same issue and since he hasn't replied I can fill in some information. All the same versions he has listed with most recent updates.

```
{
	member: {
		name: "Member"
		power: 1
		condition: "always_active"
		ftbranks.name_format: "<{name}>"
		command.checkspawns: true
		command.checkspawns.legendary: true
		
	}
	vip: {
		name: "VIP"
		power: 50
		ftbranks.name_format: "<&bVIP {name}&r>"
	}
	admin: {
		name: "Admin"
		power: 1000
		condition: "op"
		ftbranks.name_format: "<&2{name}&r>"
	}
}
```

I've tried different permissions such as pixelmon.command.checkspawns, and this is the only way that actually shows it in game as an option. But when you type it it says you must be an OP or have permission to use it even though this is supposed to be granting permission.

**Comment by desht** [comment]
@TRok92 sorry for the delay, I missed the original reply.  You've added the nodes to the `member` rank, but is the user you're testing with definitely part of that rank?  Bearing in mind that the `member` rank is the lowest-power rank, it's possible that you're in the VIP rank, for example, which doesn't have those permission nodes?

Otherwise, I'm really not sure. Your config looks valid (I presume you're not seeing errors in the server log) , and I think the node is correct, but it's hard to be certain since Pixelmon is a closed source mod and viewing the decompiled JAR isn't ideal.

**Comment by TRok92** [comment]
Yeah that's about what I was expecting. My guess is pixelmon itself just isn't recognizing permissions properly because permissions from other mods are working just fine. Thank you though!

**Comment by MichaelHillcox** [comment]
As this issue is very old and we've pushed a large amount of changes to the mod. Please check if your issue is still present in 1.21 versions of the mod. If it is, please open a new issue with updated information so it's top of our list 👍

Please do not feel obligated to follow on with this issue, we understand that situations change and we do not expect an update from any of our users.

**Comment by desht** [comment]
In this case, you need to speak with the Pixelmon authors.  As part from being on a very old version (1.16.5), Pixelmon is a closed source mod, and whatever problems are being encountered appear to be specific to Pixelmon, which suggest they have some non-standard command handling.
